formatting.css

```diff
@@ -0,0 +1,231 @@
+    width: 45%;
+}
+.header-banner .cart-alligner{
+    position: absolute;
+    top: 25%;
+    left: 85%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-size: 2em;
+}
+.cart-alligner input {
+    background-color: olivedrab;
+    color: white;
+    width: 100%;
+    font-size: 0.75em;
+    font-weight: bold;
+    height: 1.5em;
+    border: none;
+    border-radius: 4px;
+    margin-bottom: 1em;
+}
+.shopping-grid-squares .shopping-grid-squares-text button {
+    background-color: olivedrab;
+    color: white;
+    width: 100%;
+    font-size: 1em;
+    font-weight: bold;
+    height: 2em;
+    border: none;
+    border-radius: 8px;
+    margin-bottom: 1em;
+}
+.shopping-grid-squares .shopping-grid-squares-text button:hover {
+    background-color: darkgreen;
+    color: lightgray;
+    width: 100%;
+    font-size: 1em;
+    font-weight: bold;
+    height: 2em;
+    border: none;
+    border-radius: 8px;
+    margin-bottom: 1em;
+}
+#input-index {
+    background-color: lightgrey;
+    color: navy;
+    padding: 0.5em 2em;
+    font-size: 1.5em;
+    font-weight: bold;
+    height: 3em;
+    border: none;
+    border-radius: 8px;
+}
+#index-form {
+    width: 25%;
+    padding: 4em 0em 8em;
+    align-content: center;
+    transform: translate(162%, 0%);
+}
+.nav-bar {
+    width: 100%;
+    background-color: lightgray;
+    height: 4em;
+    font-size: 2em;
+    position: relative;
+    text-align: center;
+}
+.nav-bar-links{
+    color: navy;
+    position: absolute;
+    top: 50%;
+    left: 20%;
+    transform: translate(-50%, -50%);
+}
+a {
+
+}
+.footer-banner{
+    background-color: darkblue;
+}
+
+.grid-wrapper {
+    display: grid;
+    grid-template-columns: 21.7% 21.7% 21.7% 21.7%;
+    grid-gap: 4em;
+    background-color: #fff;
+    color: #444;
+    padding: 2em 0%;
+}
+.main-content{
+    padding: 0% 12%;
+}
+footer{
+    align-content: center;
+    text-align: center;
+    background-color: darkblue;
+    color: white;
+    padding-bottom: 3em;
+    font-size: 1.5em;
+}
+h1 {
+    font-size: 3em;
+    color: olivedrab;
+}
+
+h2 {
+    color: darkblue;
+    font-size: 150%;
+}
+
+h3{
+    font-size: 2.5em;
+    color: darkblue;
+}
+.shopping-grid-squares {
+    background-color: ghostwhite;
+    color: #1c1c17;
+    border-color: #cbcbbe;
+    border-width: 1em;
+    padding: 0px;
+}
+.shopping-grid-squares #item-image{
+    width: 100%;
+    height: 10em;
+}
+.shopping-grid-squares .shopping-grid-squares-text{
+    padding: 0% 10%;
+    font-size: 1.5em;
+
+}
+.cart-items {
+    width: 100%;
+    overflow: hidden;
+}
+.cart-items #descrip{
+    font-size: 1.5em;
+}
+.cart-items img{
+    height: 100px;
+    width: 100px;
+}
+.cart-items h4 {
+    font-size: 1.5em;
+    color: darkblue;
+}
+.menu-bar{
+
+}
+.menu-bar .a{
+
+}
+.blog-article-link{
+
+}
+header{
+    width: 100%;
+    padding: 0%;
+    margin: 0%;
+}
+#cart-image{
+    width: 2em;
+    height: 2em;
+    z-index: 0;

```

Comment:
you shouldn't have to use z-index in this app. We'll learn how to properly handle the background image. For now, I'd remove that img and just give the header a background color. We'll get to the background thing later on.